src/main/flight/pid.c

```diff
@@ -594,6 +621,43 @@ static void FAST_CODE applyItermRelax(const int axis, const float gyroRate, floa
         }
     }
 }
+#ifdef USE_D_BOOST
+static float FAST_CODE applyDBoost(float rateTarget, float gyroRate, flight_dynamics_index_t axis) {
```

Comment:
With all the per-axis state maintained in `pidState_t` passing `pidState_t *pidState, flight_dynamics_index_t axis` should be enough.